managed/src/main/java/com/yugabyte/yw/commissioner/tasks/EditKubernetesUniverse.java

```diff
@@ -54,6 +54,20 @@ public void run() {
 
       Universe universe = lockUniverseForUpdate(taskParams().expectedUniverseVersion);
 
+      // TODO(bhavin192): FIX ME: is this a better way? There is no
+      // way that one who writes code knows subsequent calls where
+      // this is actually needed i.e. while creating the
+      // KubernetesCommandExecutor tasks. If we go by this route, this
+      // must be verified somewhere down the call tree, so that the
```

Comment:
The code below which sets it from taskParams is ok. In our case, this is not a userIntent setting which can be changed by the user, so getting it from UI shouldn't matter, user can never edit it.

So we don't need to set it in taskParams here too, we could just pick it up from universeDetails directly.